src/software/ai/hl/stp/tactic/pass_defender/pass_defender_tactic_test.py

```diff
@@ -83,5 +83,102 @@ def test_ball_chipped_on_intercept(
     )
 
 
+@pytest.mark.parametrize(
+    "ball_initial_position,ball_initial_velocity,position_to_block_from",
+    [
+        # Place defender behind the ball and direct ball away from defender.
+        # Defender should avoid/abort intercepting the ball in these scenarios.
+        #  <---------O         X
+        #           ball    defender
+        (tbots.Point(-1, 0), tbots.Vector(-3, 0), tbots.Point(0, 0)),
+        #  <---------O    ball
+        #
+        #            X    defender
+        (tbots.Point(0, 0), tbots.Vector(-3, 0), tbots.Point(0, -1)),
+        #            ^
+        #            |
+        #            |
+        #            O    ball
+        #
+        #            X    defender
+        (tbots.Point(0, 1), tbots.Vector(0, 3), tbots.Point(0, 0)),
+    ],
+)
+def test_avoid_intercept_scenario(
+    ball_initial_position,
+    ball_initial_velocity,
+    position_to_block_from,
+    simulated_test_runner,
+):
+    # Setup Robot
+    simulated_test_runner.simulator_proto_unix_io.send_proto(
+        WorldState,
+        create_world_state(
+            [],
+            blue_robot_locations=[position_to_block_from],
+            ball_location=ball_initial_position,
+            ball_velocity=ball_initial_velocity,
+        ),
+    )
+
+    # Setup Tactic
+    params = AssignedTacticPlayControlParams()
+    params.assigned_tactics[0].pass_defender.CopyFrom(
+        PassDefenderTactic(
+            position_to_block_from=tbots.createPointProto(position_to_block_from)
```

Comment:
Does this work? Since `tbots.createPointProto` returns a C++ protobuf object, which I would suspect may be different from the same protobuf object's structure in Python

Reply:
Should be fine I think -- PassDefenderTactic is a protobuf object so we want to pass in a protobuf here (TbotsProto::Point), not Point